webapp/app/css/treeherder.css

```diff
@@ -836,36 +836,37 @@ ul.failure-summary-list li .btn-xs {
 }
 
 #pinned-job-list{
+    position: relative;
```

Comment:
I'm not sure about your use of positioning here. What's the purpose of making this relative, and the changes below to absolute? Things seem to lay out fine without these.